feature/gnoi/system/tests/complete_chassis_reboot/complete_chassis_reboot_test.go

```diff
@@ -176,3 +178,17 @@ func TestChassisReboot(t *testing.T) {
 		})
 	}
 }
+
+func FetchUniqueItems(t *testing.T, s []string) []string {
+	itemExisted := make(map[string]bool)
+	var uniqueList []string
+	for _, item := range s {
+		if _, ok := itemExisted[item]; !ok {
+			itemExisted[item] = true
+			uniqueList = append(uniqueList, item)
+		} else {
+			t.Logf("Detected duplicated item: %v", item)
```

Comment:
If this is helpful for debugging then would be nice to add some context to what is being duplicated.

Reply:
I am adding the log to collect if we found the duplicated items on different platforms. 

I will update this code when I have more info.